python/cuml/test/test_linear_model.py

```diff
@@ -400,3 +400,15 @@ def test_logistic_regression_input_type_consistency(constructor, dtype):
 
     assert isinstance(clf.predict_proba(X), original_type)
     assert isinstance(clf.predict(X), original_type)
+
+
+def test_logistic_regression_attributes():
```

Comment:
I wonder if it would be better to create a new file, `test_solver_users.py` (or something like that) and then centralize all the tests of `hasattr` there for all the classes that have a solver? Also, if we rely on the base class for coef_ and intercept_ then those would need to be added here as well.

Reply:
I thought about it, but was unsure if it was warranted. Added a new file now - testing for a few more attributes too.